Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```
@@ -3016,6 +3016,70 @@ script:
     - contextPath: Core.ScriptResult.results.retention_date
       description: Timestamp in which the retrieved files will be deleted from the server.
       type: Date
+  - arguments:
+    - description: The agent id.
+      name: agent_id
+      isArray: false
+      required: true
+    - description: The instance id.
+      name: instance_id
+      isArray: true
+      required: true
+    - description: The process name.
+      name: process_name
+      isArray: false
+      required: false
+    - description: The incident id.
+      name: incident_id
+      isArray: false
+      required: false
+    - description: For polling use.
```

```suggestion
    - description: The action ID. For polling use.
```
Is this a comma-separated list of action IDs. ?